packages/examples/src/search/includes.ts

```diff
@@ -0,0 +1,11 @@
+import { MedplumClient } from '@medplum/core';
+
+const medplum = new MedplumClient();
+
+// start-block search
```

Comment:
Two asks on this: 

1. Could you generate an example that uses `_revInclude` as well?
2. Could you generate an example response for each, here ([example](https://github.com/medplum/medplum/blob/main/packages/examples/src/search/graphql.ts#L48-L67)). ChatGPT is pretty good for this, so you don't have to generate fake data yourself